pom.xml

```diff
@@ -580,6 +580,26 @@
         <artifactId>lzma-java</artifactId>
         <version>1.3</version>
       </dependency>
+      <dependency>
+        <groupId>com.aayushatharva.brotli4j</groupId>
+        <artifactId>brotli4j</artifactId>
+        <version>1.3.0</version>
+      </dependency>
+      <dependency>
+        <groupId>com.aayushatharva.brotli4j</groupId>
+        <artifactId>native-linux-x86_64</artifactId>
+        <version>1.3.0</version>
+      </dependency>
+      <dependency>
+        <groupId>com.aayushatharva.brotli4j</groupId>
+        <artifactId>native-osx-x86_64</artifactId>
+        <version>1.3.0</version>
+      </dependency>
+      <dependency>
+        <groupId>com.aayushatharva.brotli4j</groupId>
+        <artifactId>native-windows-x86_64</artifactId>
+        <version>1.3.0</version>
+      </dependency>
```

Comment:
I think these should be optional as its an optional dependency really

Reply:
I'm not fond of having `<optional>` in `<dependencyManagement>`. IMHO, that's something to be defined on each module that use the deps. Eg, codec-http has this dep with `test` scope, no need for `optional`.
I see that defining `optional` in <dependencyManagement>` is the current coding style so I'll use it.

Reply:
Actually, not defining `<optional>` in parent's `<dependencyManagement>` but defining it in the `child` that requires this dep is consistent with how things are done for all the other codecs, eg https://github.com/netty/netty/blob/4.1/pom.xml#L578-L582 vs https://github.com/netty/netty/blob/4.1/codec/pom.xml#L81-L85.

What would you rather have?

Reply:
sounds good